File: Saummya/LINKEDLIST/GFG/Occurence of an integer in a Linked List.py
"""  
class Node:
    def __init__(self, data):
		self.data = data
		self.next = None
  This is method only submission.
  You only need to complete the method.
"""
class Solution:
    def count(self, head, search_for):
        # Code here
        count=0
        head1=head
        while head1:
            while head1.data==search_for:
                count+=1
            head1=head1.next
            
        return count

File: Vivek/Linked List/Occurence of an integer in a Linked List.c
/*Node is defined as 
struct node
{
    int data;
    struct node* next;
    
    node(int x){
        data = x;
        next = NULL;
    }
}*head;
*/
class Solution
{
    public:
    int count(struct node* head, int search_for)
    {
      struct node*ptr = head;
      int count=0;
      while(ptr!=NULL)
      {
          if(ptr->data==search_for)
            count++;
        ptr=ptr->next;
      }
    return count;
    } 
};
